### ISSUE TITLE
Preview Method not working

### ISSUE THREAD

**Issue by Weaze1** [ClownsharkBatwing/RES4LYF#31]
The ClownsharkKsampler doesnt show the process of sampel/unsampeling even tho the encoder is set to TAESED/Latent2RGB/Auto via Comfyui Manager. After generating one image with fixed seed, setting the preview method from Auto to TAESED (for example) and only then the preview is working. It never works within the first generation. I always need to sample one image at least twice in order to see what the sampler is doing. Why is there no Preview Method  Option (TAESED/Latent2RGB/Auto) within the ClownsharkKsampler. This would at least allow to switch the Preview Method again without needing to go into the Manager and choose it there. would be one step of annoyance less. But the fact that the sampler isnt showing the generation at firstplace its very frustrating when you pipe a normal sampler via latent to the ClownsharkKsampler. Is this sampler only for "Load Image" ?

**Comment by ClownsharkBatwing** [comment]
The issue might be resolved now, if your problem was with TAESD. I do recommend using Latent2RGB though as it's much faster and pretty much gives you the same information.

No sampler that I'm aware of has a preview method option directly within the node itself, that's handled by ComfyUI.

What do you mean by "pipe a normal sampler via latent"? Do you mean you're doing img2img with a "normal sample" or are you trying to use some other sampler other than ClownSampler for some reason?

**Comment by Weaze1** [comment]
You are right. there is only a switch for preview on/off in other samplers. No option for chaning the encoder method. 
Yes i mean a new generated image (for example of the standart workflow) into ClownsharkKsampler which does the 2nd pass set to unsampling (into ClownsharkKsampler resample for 3rd Pass. Latent Output to Latent Input. Not from "Load Image node". Sampler to Sampler. I tryed to use "preview bridge node" as "load Image replacement" but without success. 
Im really looking forward to your fix. I appreciate your work. That "eta option" is completly amazing. i am sure i can find out a lot more which i like just let me see the progress.

**Comment by Weaze1** [comment]
Oh maybe you fixed it for SD3.5 yea. I used SDXL1.0.

**Comment by ClownsharkBatwing** [comment]
Gotcha, so is the issue that it doesn't work at all? Or does it work, just without showing the preview?

**Comment by Weaze1** [comment]
Just without showing the preview but when you hit a specific point in unsampling mode the resampler does magic. Im addicted.
Thats why i would love the see the process from beginning and not have to "fix seed/change method/que again.
But even with this torture of reque this sampler is the best improvement in utility of comfyui.

**Comment by ClownsharkBatwing** [comment]
Any chance you could provide a workflow (ideally, something with only RES4LYF and default ComfyUI nodes) that can demonstrate the problem? A screenshot would be sufficient.

**Comment by Weaze1** [comment]
After a fresh install of Comfyui i tryed to reproduce my experience described before. Using the "Standart Comfyui Workflow"  in combination of "RES4LYF img2img unsampling SDXL output workflow".  See below. I could NOT reproduce the error i experienced. Back to my Main Installation of Comfyui and workflow instead of replacing clownsharkKsampler and seeing TEASED working i replaced my first Sampler this time which feeds clownsharkKsampler and I see its working perfectly. I have to admit that the clownsharkKsampler isnt the problem. I was using Ksampler (Efficent) and this reproduces the Error i experienced. Maybe u can find out whats happening there. Ima sorry for the trouble. I used the clownsharksampler even with no preview because i just love it. Please keep it up and thanks for response.
![KsamplerEfficent](https://github.com/user-attachments/assets/18a893bb-0544-4785-8300-5461f2101adc)
![KsamplerEfficent2](https://github.com/user-attachments/assets/653459dd-d453-4597-96fa-22b834ce2107)
![KsamplerNormal](https://github.com/user-attachments/assets/7baa9f35-5511-49ed-ad99-9307ea6ca140)
![KsamplerNormal2](https://github.com/user-attachments/assets/fb39107d-aa78-41dc-a929-a43a95176792)
